reactDashboard/src/components/GeoHeatMap/GeoHeatMap.jsx

```diff
@@ -156,17 +156,17 @@ const GeoHeatMap = ({
 
       if (regionData) {
         content =
-          "<div class='region-popup'><div><strong>" +
+          "<div class='region-popup'><div>" +
```

Comment:
run prettier on this file

Reply:
Which part? These elements are string literals, so prettier won't do anything

Reply:
Just ignore my comment then :D resolved :D

Reply:
✅